omod/src/main/webapp/locationMapping.jsp

```diff
@@ -0,0 +1,61 @@
+<%@ include file="/WEB-INF/template/include.jsp" %>
+<%@ include file="/WEB-INF/template/header.jsp" %>
+
+<openmrs:htmlInclude file="/moduleResources/dhisconnector/flexboxgrid.min.css"/>
+<openmrs:htmlInclude file="/moduleResources/dhisconnector/dhisconnector.css"/>
+
+<%@ include file="template/localHeader.jsp" %>
+
+<c:if test="${showLogin == 'true'}">
+    <c:redirect url="../../login.htm"/>
+</c:if>
+
+<h3><spring:message code="dhisconnector.locationMapping"/></h3>
+
+<form method="post">
+    <table id="locationMappingTable">
+        <thead>
+        <tr>
+            <th><spring:message code="dhisconnector.locationMapping.location"/></th>
+            <th><spring:message code="dhisconnector.locationMapping.orgUnit"/></th>
+        </tr>
+        </thead>
+
+        <tbody>
+        <c:forEach items="${locations}" var="location">
+            <tr class="evenRow">
+
```

Comment:
Let's temove unwanted spaces.

Reply:
Okay. I'll update the PR.

Reply:
Fixed with [041fd7b](https://github.com/openmrs/openmrs-module-dhisconnector/commit/041fd7b82d787717170bb3a0acb07c859cc76fd9)